.github/workflows/pr_ping.yml

```diff
@@ -5,26 +5,40 @@ on:
   pull_request_target:
     types:
     - opened
+    - ready_for_review
 
 jobs:
   send_message:
-    name: Send message
+    name: Send Slack message
     if: |
       github.actor != 'dependabot[bot]' &&
       !startsWith(github.event.pull_request.title, '🔄')
     runs-on: ubuntu-latest
+    env:
+      slack_url: ${{ secrets.SLACK_WEBHOOK_URL }}
+      pr_url: ${{ github.event.pull_request.html_url }}
+      pr_number: ${{ github.event.pull_request.number }}
+      pr_title: ${{ github.event.pull_request.title }}
+      pr_author: ${{ github.event.pull_request.user.login }}
+      pr_repo: ${{ github.event.pull_request.base.repo.full_name }}
     steps:
-    - name: Send Slack notification
+    - name: Send notification for new PR
+      if: github.event.action == 'opened'
+      env:
+        pr_status: ${{ github.event.pull_request.draft && ' draft ' || ' ' }}
+        pr_icon: ${{ github.event.pull_request.draft && ':pr-draft:' || ':pull-request:' }}
```

Comment:
[Reference](https://github.community/t/possible-to-use-conditional-in-the-env-section-of-a-job/135170/2); alternatively, we can also set env vars [using a `run` step](https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#setting-an-environment-variable)